docs/src/pages/components/popover/popover.md

```diff
@@ -28,7 +28,7 @@ the position of the popover.
 
 ## Mouse over interaction
 
-This demonstrates how to use the `Popover` component to implement a popover behavior based on the mouse over event.
+This demonstrates how to use the `Popover` component to implement a popover behavior based on the mouseover event.
```

Comment:
I know this was previously used but 'to implement a popover behaviour' doesn't sound right to me. 

I think something like this sounds much better:
```suggestion
This demonstrates how to use the `Popover` component and the mouseover event to achieve popover behavior.
```

Reply:
Yep definitely makes more sense ~